src/Symfony/Component/Cache/Adapter/TagAwareAdapter.php

```diff
@@ -33,11 +33,14 @@ class TagAwareAdapter implements TagAwareAdapterInterface, PruneableInterface, R
     private $getTagsByKey;
     private $invalidateTags;
     private $tags;
+    private $knownTagVersions = array();
+    private $knownTagVersionsTtl;
 
-    public function __construct(AdapterInterface $itemsPool, AdapterInterface $tagsPool = null)
+    public function __construct(AdapterInterface $itemsPool, AdapterInterface $tagsPool = null, $knownTagVersionsTtl = 0.15)
     {
         $this->pool = $itemsPool;
         $this->tags = $tagsPool ?: $itemsPool;
+        $this->knownTagVersionsTtl = $knownTagVersionsTtl;
```

Comment:
How should we use this property when we always want to fetch the tag versions?

Reply:
doesn't `0` work? or is the check [L351](https://github.com/symfony/symfony/pull/27007/files#diff-2fe2f12a84197b6987fbd5e133cd09aeR351) wrong? PR welcome if you found a bug.

Reply:
0 didn't fix it for me, seems like the check is wrong.
Filling in 1000 did work.
I'll look into it a bit better.

Reply:
I've made a PR for this #33754